test/parallel_tests/helpers.py

```diff
@@ -0,0 +1,57 @@
+import numpy
+
+
+def get_tmpdir(comm):
```

Comment:
This is correct, but incomplete:
the content of `/tmp` is written in virtual memory. In case of distributed processes over multiple nodes, the processes will not all have access to the same temporary directory.

Would be good to do the following:
1) after the broadcast check if all processes can access the temporary directory
2) if not, raise an error saying to set a different temporary location using `TMPDIR`, `TEMP` or `TMP` enviroment variables which are used by `tempfile` for determining the temporary directory: https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir